src/fides/api/api/deps.py

```diff
@@ -29,8 +29,14 @@ def get_db() -> Generator:
 
 
 @contextmanager
-def get_db_contextmanager() -> Generator[Session, None, None]:
-    """Return our database session as a context manager"""
+def get_autoclose_db_session() -> Generator[Session, None, None]:
+    """
+    Return a database session that automatically closes when the context exits.
```

Comment:
```suggestion
    Return a database session as a context manager that automatically closes when the context exits.
```

just so we don't lose the fact that it's a context manager from the docstring